processor-steps/src/main/java/boringyuri/processor/common/steps/AssociatedUriDataGeneratorStep.kt

```diff
@@ -104,21 +118,25 @@ class AssociatedUriDataGeneratorStep(
         // Iterating over method parameters we'll find all the replacements for
         // the method templates found on the previous step and create the params list.
         methodParameters.forEach { param ->
-            val paramName = param.simpleName.toString()
-            val defaultValue = param.getAnnotation<DefaultValue>()?.value
+            val paramName = param.name
+            val defaultValue = param.getAnnotation<DefaultValue>()
```

Comment:
nit: looks like you may keep this line unchanged
```suggestion
            val defaultValue = param.getAnnotation<DefaultValue>()?.value
```
this way the other three lines below won't need to be changed.

Reply:
done